### ISSUE TITLE
arXiv-2019/05-Unified Language Model Pre-training for Natural Language Understanding and Generation

### ISSUE THREAD

**Issue by BrambleXu** [BrambleXu/knowledge-graph-learning#205]
**一句话总结：**

即BERT之后的另一种LM。与BERT不同，UNILM能进行细节上的设置，使用不同的self-attention mask.

> 问题：
> 提案：
> 具体做法：
> 效果：

**资源：**

- [pdf](https://arxiv.org/abs/1905.03197)
- [code](
- [paper-with-code](

**论文信息：**

- Author: Microsoft Research
- Dataset: 
- keywords: 

**笔记：**

![image](https://user-images.githubusercontent.com/10768193/58619909-cf241d80-8300-11e9-8a4c-6ef53f736728.png)

![image](https://user-images.githubusercontent.com/10768193/58619897-caf80000-8300-11e9-942e-43ff3e6765d7.png)

Similar to BERT, the pre-trained UNILM can be fine-tuned (with additional task-specific layers
if necessary) to adapt to various downstream tasks. But unlike BERT which is used mainly for natural language understanding tasks, UNILM can be configured, using different self-attention masks which will be detailed in Section 2, to aggregate context for different types of language models, and thus can be used for both natural language understanding and generation tasks.

**模型图：**

![image](https://user-images.githubusercontent.com/10768193/58620036-17434000-8301-11e9-99b3-f98756f42158.png)

**结果**：

![image](https://user-images.githubusercontent.com/10768193/58620065-25915c00-8301-11e9-8644-58b5ff9ab9b0.png)

**接下来要看的论文**：
